<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialCOM</class>
 <widget class="QMainWindow" name="SerialCOM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>467</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Serial COM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/images/wing_com.png</normaloff>:/images/wing_com.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	color: rgb(255, 255, 255);
	background-color: rgb(215, 214, 255);
	background-repeat: no-repeat;
	background-position: left top;
	background-origin: content;
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(112,128,144, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_PlainText">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="horizontalSpacing">
            <number>12</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Port Name:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Open Mode:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_DataBits">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	combobox-popup: 0;
	background-color: rgb(119,136,153);
	color: rgb(240, 255, 255);
}
</string>
             </property>
             <property name="currentIndex">
              <number>3</number>
             </property>
             <property name="maxVisibleItems">
              <number>3</number>
             </property>
             <property name="maxCount">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>Data5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Data6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Data7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Data8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Flow Control:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboBox_OpenMode">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	combobox-popup: 0;
	background-color: rgb(119,136,153);
	color: rgb(240, 255, 255);
}</string>
             </property>
             <property name="maxVisibleItems">
              <number>3</number>
             </property>
             <property name="maxCount">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>Read Only</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Write Only</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Read Write</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Baud Rate:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Data Bits:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_PortList">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	combobox-popup: 0;
	background-color: rgb(119,136,153);
	color: rgb(240, 255, 255);
}</string>
             </property>
             <property name="maxVisibleItems">
              <number>5</number>
             </property>
             <property name="maxCount">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Parity:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QComboBox" name="comboBox_FlowControl">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	combobox-popup: 0;
	background-color: rgb(119,136,153);
	color: rgb(240, 255, 255);
}</string>
             </property>
             <property name="maxVisibleItems">
              <number>3</number>
             </property>
             <property name="maxCount">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>No Flow Ctrl</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hardware Ctrl</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Software Ctrl</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QComboBox" name="comboBox_Parity">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	combobox-popup: 0;
	background-color: rgb(119,136,153);
	color: rgb(240, 255, 255);
}</string>
             </property>
             <property name="maxVisibleItems">
              <number>3</number>
             </property>
             <property name="maxCount">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>No Parity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Even Parity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Odd Parity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Space Parity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mark Parity</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QComboBox" name="comboBox_BaudRate">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	combobox-popup: 0;
	background-color: rgb(119,136,153);
	color: rgb(240, 255, 255);
}</string>
             </property>
             <property name="currentIndex">
              <number>3</number>
             </property>
             <property name="maxVisibleItems">
              <number>3</number>
             </property>
             <property name="maxCount">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>1200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox_StopBits">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox{
	combobox-popup: 0;
	background-color: rgb(119,136,153);
	color: rgb(240, 255, 255);
}</string>
             </property>
             <property name="maxVisibleItems">
              <number>3</number>
             </property>
             <property name="maxCount">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>One Stop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Two Stop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>One &amp; Half Stop</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Stop Bits:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" colspan="2">
            <widget class="QPushButton" name="pushButton_Open">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(119,136,153);
color: rgb(240, 255, 255);</string>
             </property>
             <property name="text">
              <string>Open</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_Clear">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(119,136,153);
color: rgb(240, 255, 255);</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_Hex">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Hex</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Send">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(119,136,153);
color: rgb(240, 255, 255);</string>
            </property>
            <property name="text">
             <string>SEND</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(119,136,153);
color: rgb(240, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Save">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(119,136,153);
color: rgb(240, 255, 255);</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Us"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </widget>
  <action name="actionAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
